src/core/tests/runtime/interpreter/unit_test.manifest

```diff
@@ -140,3 +140,7 @@ INTERPRETER.onnx_dyn_shapes_flatten_neg_axis
 INTERPRETER.onnx_dyn_shapes_slice_10_slice_2d_default_steps_dyn_begin_end
 INTERPRETER.onnx_model_instance_normalization_dyn_shape
 INTERPRETER.onnx_controlflow_loop_2d_no_identity_termination_cond_false
+
+# Segmentation fault (core dumped)
```

Comment:
Do you have a stacktrace? I think we need to fix it or create a ticket

Reply:
So I was able to fix the problem, and I was trying to avoid not necessary changes.
Please review: 
`docs/template_plugin/backend/int_executable.cpp` and `docs/template_plugin/src/template_executable_network.cpp`